test/Microsoft.AspNet.Mvc.Core.Test/VirtualFileResultTest.cs

```diff
@@ -241,73 +267,29 @@ public void Constructor_SetsContentTypeAndParameters()
             Assert.Equal(expectedData, contents);
         }
 
-        [Theory]
```

Comment:
I agree that you've got tests which verify the `path` is passed unchanged (or with tilde removed) to the underlying `IFileProvider` in a few simple cases. What I'm asking is a few more cases added to the `ExecuteResultAsync_ReturnsFiles_ForDifferentPaths()` and perhaps `ExecuteResultAsync_TrimsTilde_BeforeInvokingFileProvider()` test data. That is, the new pass-through behaviour isn't tested enough.